src/phoenix/session/session.py

```diff
@@ -423,6 +426,22 @@ def get_evaluations(
         return project.export_evaluations()
 
 
+def reset_all(hard: Optional[bool] = False) -> None:
+    """
+    Resets everything to the initial state.
+    """
+    working_dir = get_working_dir()
+
+    # See if the working directory exists
+    if working_dir.exists():
+        if not hard:
+            input(
+                f"You have data at {working_dir}. Are you sure you want to delete?"
+                + " This cannot be undone. Press Enter to delete."
```

Comment:
Users may be confused here because thy can't say no.

Reply:
Added Escape to cancel.